VimeoNetworking/VimeoNetworking/AuthenticationController.swift

```diff
@@ -160,38 +200,75 @@ final public class AuthenticationController
     
     // MARK: - Private Authentication
     
+    /**
+     **(PRIVATE: Vimeo Use Only)**
+     Log in with an email and password
+     
+     - parameter email:      a user's email
+     - parameter password:   a user's password
+     - parameter completion: handler for authentication success or failure
+     */
```

Comment:
I think it might be worth adding something like `- Warning: This method will not work for third party applications.` to all of these private Vimeo methods. Or at least some way to indicate that it's not just that we don't want people to use it. 

It's that they _can't_.